messaging/relayer/deploy/main.tf

```diff
@@ -0,0 +1,145 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.5.0"
+    }
+  }
+  required_version = ">= 0.13"
+}
+
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+resource "aws_instance" "relayer" {
+  ami           = "ami-0567f647e75c7bc05"
+  instance_type = "t2.small"
+  tags = {
+    Name = "relayer"
+  }
+  security_groups = ["security_relayer"]
+  user_data       = <<-EOF
+    #!/bin/sh
+    cd /home/ubuntu/
+    sudo apt-get update
+    sudo apt install -y unzip git make build-essential ca-certificates curl gnupg lsb-release
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+    echo \
+        "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update
+    sudo apt-get install -y docker-ce docker-ce-cli containerd.io
```

Comment:
we need to move this to a Docker instance and publish an image. 
do you have access to any harbor repo? if no, I will ask for devops so we can have one repo to publish our private images (for now)